frontend/javascript/components/postupload/post-upload.vue

```diff
@@ -0,0 +1,1663 @@
+                DEBUG set today
+              </button>
+            </template>
+            <template v-else>
+              <button
+                type="button"
+                @click="gotoStep()"
+                :disabled="!gotoValid"
+                class="btn btn-primary d-block w-100">
+                weiter
+              </button>
+            </template>
+          </div>
+          <!--/.col-lg-9-->
+        </div>
+        <!--/.row-->
+      </div>
+      <!--/.container-->
+    </div>
+    <!--/.sticky-bottom-->
+  </div>
+</template>
+
+<style lang="scss" scoped>
+@use 'sass:map';
+@import 'bootstrap/scss/functions';
+@import '../../../styles/variables.scss';
+
+.debug {
+  opacity: 0.5 !important;
+  color: #888 !important;
+
+  &.btn {
+    color: black !important;
+  }
+}
+
+/* stretch "main" so actions-footer always sticks to the bottom,
+ * giving it an app-like feel */
+
+.postupload-main {
+  // header
+  //   #header nav's padding
+  //   #header nav .navlogo svg's height
+  //   .breadcrumb's padding
+  //   breadcrumb-item's line-height
+  // footer (sticky)
+  //   py-2
+  //   btn's padding
+  //   btn's height
+  min-height: calc(
+    100vh - (2 * 1rem + 2.5rem) - (2 * 0.5rem + 1.5rem * 14 / 16) -
+      (2 * 0.5rem + 2 * 0.375rem + 1.5rem)
+  );
+}
+
+/* make form-check more accessible; whole block is padded + clickable */
+
+.form-check {
+  background-color: var(--bs-tertiary-bg);
+  padding: 0;
+  margin-top: 0.5em;
+  margin-bottom: 0.5em;
+
+  &:hover {
+    background-color: var(--bs-secondary-bg);
+  }
+}
+
+.form-check label {
+  padding: 1em 1em 1em 2.5em;
+  display: block;
+}
+
+.form-check-input {
+  margin-top: 1.25em;
+  margin-left: 0.75em;
+  border-color: var(--bs-body-color);
+  border-width: 2px;
+
+  .form-check:hover & {
+    border-color: var(--bs-primary);
+  }
+
+  &:checked {
+    background-color: black;
+  }
+}
+
+/* make links stand out more in alert boxes */
+
+.alert {
+  a {
+    text-decoration: underline;
+  }
+}
+
+/* re-style the component's filename-dialog so first it looks less interactive,
+ * until you click "change filename" */
+
+.documents-filename {
+  max-width: 20em;
+  --bs-link-color-as-light-bg: #0047e120;
+  background-color: var(--bs-link-color-as-light-bg);
+
+  input {
+    font-weight: bolder;
+  }
+
+  /* "hide" that this is a regular, editable Bootstrap input... */
+  input:not(:focus) {
+    background: transparent;
+    border-color: transparent;
+    box-shadow: none;
+  }
+
+  /* ...until its label (looking like a link-button) is clicked, which gives it focus */
+  label {
+    cursor: pointer;
+    text-decoration: underline;
+    color: var(--bs-link-color);
+    /* padding-left matches bootstrap's form-group>input for indent */
+    padding: 0.25rem 0.25rem 0.25rem 0.75rem;
+  }
+}
+
+/* bootstrap doesn't provide this responsive position class out of the box
+ * (it could be compiled in with utils/helpers)
```

Comment:
this can be configured: https://getbootstrap.com/docs/5.3/utilities/api/#using-the-api

Reply:
done! wasn't sure if this one case was worth the overhead, but also hadn't seen that we already have utilities configured.. thx